components/Observations.tsx

```diff
@@ -83,26 +83,32 @@ export const ObservationSummaryCard: React.FunctionComponent<{
   const navigation = useNavigation<HomeStackNavigationProps>();
   const {width} = useWindowDimensions();
   return (
-    <>
-      <WingBlank size={'md'}>
-        <TouchableOpacity
-          onPress={() => {
-            navigation.navigate('observation', {
-              id: observation.id,
-            });
-          }}>
-          <Card>
-            <Card.Header title={zone + ' - ' + observation.locationName} extra={observation.startDate} />
-            <Card.Body>
-              <View style={{padding: 10}}>
-                <RenderHTML source={{html: observation.observationSummary}} contentWidth={width} />
-              </View>
-            </Card.Body>
-            <Card.Footer content={observation.name} extra={observation.observerType} />
-          </Card>
-        </TouchableOpacity>
-      </WingBlank>
-      <WhiteSpace size={'sm'} />
-    </>
+    <Box px="2" pt="2">
+      <TouchableOpacity
+        onPress={() => {
+          navigation.navigate('observation', {
+            id: observation.id,
+          });
+        }}>
+        <Box bg="white" borderWidth="2" borderRadius="8" borderColor="light.200" p="4">
+          <VStack space="2">
+            <HStack justifyContent="space-between" alignItems="center">
```

Comment:
Is `HStack` basically just a flexbox presenting a column? Time will tell but I hope doing things this way doesn't make it hard to get the full flexbox set of possibilities.

Reply:
FWIW I really like how the flex-box things are front-and-center in the TSX here ... it was really weird to have them in the style for vanilla React Native, stuff that was super critical to the view ending up very low down on the page.

Reply:
> Is HStack basically just a flexbox presenting a column? Time will tell but I hope doing things this way doesn't make it hard to get the full flexbox set of possibilities.

It's basically that, but then adds a `space` prop. So you can say "I want to stack these things in a column with 2px between them", which turns out to be a very useful way to build a lot of UI as you nest things together. We used this pattern at Stripe for lots of our UI.

Reply:
Totally agree with your assessment of the styles being separated from the components - very hard to make changes.